### PR TITLE
[Find Forms] Add E2E test

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR adds an E2E test for `/find-forms`.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/6473

## Testing done
Ran the test locally via `npm run watch:static` in one terminal and `npm run test:e2e -- src/applications/find-forms/tests/00-required.e2e.spec.js` in another.

## Screenshots
N/A

## Acceptance criteria
- [x] E2E test is written

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/find-forms/tests/00-required.e2e.spec.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      browser.assert.elementPresent(`a[href="${form.attributes.url}"]`);
+    });
+
+    const nextPage = pageNumber + 1;
+    const hasNextPage = nextPage < pages.length;
+
+    if (hasNextPage) {
+      browser.click(SELECTORS.NEXT_PAGE);
+    }
+  });
```
Comment (ncksllvn) [inline src/applications/find-forms/tests/00-required.e2e.spec.js:51]: This loop is powerful in that it confirms there are links to every form in the JSON response, but the sort and page-size are directly tied to the FE, so I'm not sure if it's worth making it more agnostic to the FE implementation.

I thought about selecting the form links from the DOM and comparing the HREFs to values in the JSON, but the Nightwatch docs don't have a clear way to grab an array of elements. `.elements()` didn't seem to work. - http://v09.nightwatchjs.org/api/

**Comment by ncksllvn** [comment]
image

Ah, I forgot we can't use `async`/`await` during E2E tests for some reason.

**Comment by ncksllvn** [comment]
Still looking for a review on this one if someone has a moment!

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

**Pull request merged**